test/e2e/peered/node.go

```diff
@@ -119,20 +120,26 @@ func (c NodeCreate) Create(ctx context.Context, spec *NodeSpec) (PeerdNode, erro
 		EKSEndpoint:       spec.EKSEndpoint,
 		KubernetesVersion: spec.NodeK8sVersion,
 		NodeadmUrls:       c.NodeadmURLs,
+		NodeadmConfig:     nodeadmConfig,
 		NodeadmConfigYaml: string(nodeadmConfigYaml),
```

Comment:
what's the point of passing these two?

Reply:
Here I'm just using the `NodeadmConfig` field of type `api.NodeConfig` which is a part of the `UserDataInput` struct. Bottlerocket also consumes several fields present in the node config such as SSM primitives such as activation ID/activation code, IAM RA primitives such as cert/key/trust anchor/profile etc, just that it doesn't consume them by running `nodeadm`, but the underlying usage is the same so I left them here. The `NodeadmConfigYaml` field is unused by Bottlerocket but used by other OS's like RHEL, Ubuntu, AL2023 which actually run `nodeadm`.